gcovr/tests/test_args.py

```diff
@@ -103,7 +103,7 @@ def test_line_threshold_100_1(capsys):
 def test_filter_backslashes_are_detected(capsys):
     c = capture(
         capsys,
-        args=['--filter', r'C:\\foo\moo', '--gcov-exclude', ''],
```

Comment:
Couldn't really figure out the necessity of this argument as it seems to have no effect or relation at all to the actual test.

Reply:
I now understand why `--gcov-exclude` was used here. The `capture()` test driver runs gcovr's main(). It actually searches for coverage data and creates a report. But this test is not about the reports, only about side effects. Setting `--gcov-exclude` to an empty filter ensures that no coverage data will be found, which should make the test more robust. The empty filter is the easiest way to write a filter that always matches, here: excludes all files.